.github/actions/BuildCirct/action.yml

```diff
@@ -30,5 +30,6 @@ runs:
       run: |
         ./scripts/build-circt.sh \
           --build-path ${{ github.workspace }}/build-circt \
-          --install-path ${{ github.workspace }}/build-circt/circt
+          --install-path ${{ github.workspace }}/build-circt/circt \
```

Comment:
This is minor, but the MLIR installs into usr instead of into the build directory.

Reply:
Yes, but this is unrelated to this PR. We can change this if wanted, but not in this PR. I just want to get this PR through as all progress is stopped because of it.